src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/common.kt

```diff
@@ -465,6 +467,10 @@ fun <T> runPipelineWithDeadLetterQueue(
           logger.info(
             "Event: [${eventPayload.toString(StandardCharsets.UTF_8)}] successfully sent with visibility timeout: [${it.value.timeNextVisible}] ms to queue: [${deadLetterQueueAsyncClient.queueName}]")
         }
+        .doOnNext {
+          logger.info(
+            "Event: [${eventPayload.toString(StandardCharsets.UTF_8)}] successfully sent with visibility timeout: [${it.value.timeNextVisible}] ms to queue: [${deadLetterQueueAsyncClient.queueName}]")
+        }
```

Comment:
duplicated doOnNext?

Reply:
Fixed in Fixed in 5ceb643730d965324fb2d6321260788bdee1c44b